### DISCUSSION TITLE
extracting recording date & time from its

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#323] [Q&A]
I think there is a function in metadata import functions from EL1000 that would be useful within ChildProject, namely the one that extracts date & time from .its.

(Perhaps it IS already there & I forgot?)

@MarvinLvn https://gin.g-node.org/LAAC-LSCP/phonSES/issues/1

**Comment by MarvinLvn** [comment]
Renaming has been done.
Not sure where I should look for metadata import functions ... EL1000 : https://github.com/LAAC-LSCP/EL1000
does not contain any code, only issues.

**Comment by MarvinLvn** [comment]
https://gin.g-node.org/EL1000
https://gin.g-node.org/EL1000/el1000
https://github.com/LAAC-LSCP/EL1000

:/

**Reply by lucasgautheron** [reply]
Hello,

here it is: https://gin.g-node.org/laac-lscp/tools/src/master/EL1000/metadata.py#L93

**Reply by lucasgautheron** [reply]
Note: it's not going to do that for you, so you'd still need to write a script for it.

I can just do it myself. I can also add this as a feature of ChildProject, as @alecristia suggested.
What do you want @MarvinLvn  ?

**Reply by MarvinLvn** [reply]
I think adding this as a feature for ChildProject would be very useful (just the way we add audio durations :))
I'm volunteering to try it first !
